framework/wfa/ucc-dummy.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import socket
+import argparse
+import sys
+import threading
+import binascii
+
+def handle(host, port, respDict):
+    id = threading.get_ident()
+    print("{}: Listening on {}:{}".format(id, host, port))
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind((host, port))
+        s.listen()
+        conn, addr = s.accept()
+        with conn:
+            print("{}: {}@{} Connected by {}".format(id, host, port, addr))
+            while True:
+                try:
+                    data = conn.recv(1024)
+                except Exception as e:
+                    print("Exception {}".format(e))
+                    break
+                print("{}@{} Received: {}".format(host, port, data))

```

Comment:
Ideally, you should buffer incoming data and split on `\r\n` because the TCP stream has no packet boundaries. However, since UCC will not send a new command until it received a response from the previous one, it will work as is. So OK.